CHANGELOG.md

```diff
@@ -2,6 +2,60 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.32.0](https://github.com/serverless/serverless/compare/v2.31.0...v2.32.0) (2021-03-30)
+
+### Features
+
+- **AWS HTTP API:**
+  - Add support for AWS IAM authorization ([#9195](https://github.com/serverless/serverless/pull/9195)) ([d3c6e43](https://github.com/serverless/serverless/commit/d3c6e4323b9a3345d71ec43e6ac3013c0ffa02b7)) ([Piotr Grzesik](https://github.com/pgrzesik))
+  - Add support for custom Lambda authorizers ([#9192](https://github.com/serverless/serverless/pull/9192)) ([37d03b6](https://github.com/serverless/serverless/commit/37d03b6888788b2ee0b4a679de30ba94d25d7d53)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **AWS IAM:** Add support for `iam.role.name` definition ([#9166](https://github.com/serverless/serverless/issues/9166)) ([8c3e1be](https://github.com/serverless/serverless/commit/8c3e1be735120f5e49f6850259072c13e175b71f)) ([Sergii Kovalev](https://github.com/Enase))
+- **AWS Lambda:** Do not require all `image` properties ([#9177](https://github.com/serverless/serverless/pull/9177)) ([14f5743](https://github.com/serverless/serverless/commit/14f57438467b5d5c9a3dc356ac4a8b0a6021657f)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Validate command and options against resolved schema ([#9171](https://github.com/serverless/serverless/pull/9171)) ([2dacbcc](https://github.com/serverless/serverless/commit/2dacbcce8529378d94abc7a0b0d0039ecee4d790)) ([Mariusz Nowak](https://github.com/medikoo))
+- **Packaging:** Deprecate `include` & `exclude` in favor of `patterns` ([#8581](https://github.com/serverless/serverless/issues/8581)) ([e1678fb](https://github.com/serverless/serverless/commit/e1678fb1c65ab0246e60d44857466be6771f889d)) ([Juanjo Diaz](https://github.com/juanjodiaz))
+- **Variables:**
+  - Recognize CLI command plugin extensions in new resolver ([#9171](https://github.com/serverless/serverless/pull/9171)) ([3422a12](https://github.com/serverless/serverless/commit/3422a121d787fa716f708051f0bfe97644a3c1aa)) ([Mariusz Nowak](https://github.com/medikoo))
```

Comment:
This one belongs to `CLI` scope (sorry, by mistake I've commited it with `Variables` scope)